wasi/wasi.go

```diff
@@ -33,18 +31,6 @@ func InstantiateSnapshotPreview1(r wazero.Runtime) (api.Module, error) {
 }
 
 const (
-	// functionStart is the name of the nullary function a module must export if it is a WASI Command Module.
```

Comment:
this diff changed no behavior, just moved the docs to the struct types and removed the interface